Revise README for FocalPointPicker component to use object-position (#77722)
WordPress/gutenberg: packages/components/src/focal-point-picker/README.md

Reviewer: Whatever changes we end up making to the README,  we should always keep them in sync with the JSDoc in `index.tsx`

# FocalPointPicker

> Focal Point Picker is a component which creates a UI for identifying the most important visual point of an image. It addresses two common issues in responsive image rendering. First, large background images are often cropped in undesirable ways, especially on smaller viewports such as mobile devices. Second, the CSS aspect-ratio property can inadvertently crop out the area of highest visual interest. This component allows the selection of the point with the most important visual information and returns it as a pair of numbers between 0 and 1. The output value can be applied to either CSS `background-position` (for elements with `background-image`) or `object-position` (for `<img>` / `<video>` elements rendered with `object-fit: cover`).

Reviewer: Keeing in sync with previous suggestion:

```suggestion
Focal Point Picker is a component which creates a UI for identifying the most important visual point of an image.

It addresses two common issues when displaying images in cropped containers. First, large background images can be cropped in undesirable ways, especially on smaller viewports such as mobile devices. Second, the CSS aspect-ratio property can inadvertently crop out the area of highest visual interest. This component allows the selection of the point with the most important visual information and returns it as a pair of numbers between 0 and 1. The output value can be applied to either CSS `background-position` (for elements with `background-image`) or `object-position` (for `<img>` / `<video>` elements rendered with `object-fit: cover`).
```

-   Example focal point picker value: `{ x: 0.5, y: 0.1 };`

> -   Corresponding CSS: `object-position: 50% 10%;`

Reviewer: As above

```suggestion
-   Example focal point picker value: `{ x: 0.5, y: 0.1 }`;
-   Corresponding CSS: `object-position: 50% 10%`;
```

# FocalPointPicker

Focal Point Picker is a component which creates a UI for identifying the most important visual point of an image. It addresses two common issues in responsive image rendering. First, large background images are often cropped in undesirable ways, especially on smaller viewports such as mobile devices. Second, the CSS aspect-ratio property can inadvertently crop out the area of highest visual interest. This component allows the selection of the point with the most important visual information and returns it as a pair of numbers between 0 and 1. This value can be easily converted into the CSS `object-position` attribute, and will ensure that the focal point is never cropped out, regardless of viewport.

- Example focal point picker value: `{ x: 0.5, y: 0.1 }`

> - Corresponding CSS: `object-position: 50% 10%`

Reviewer: Let's add trailing semi-columns for both bullet points.

				onDrag={ setFocalPoint }
				onChange={ setFocalPoint }
			/>

> 			<img src={ url } style={ style }  />

Reviewer: I don't think the current example snippet will work as expected, we need to set `object-fit` and also ideally a target ratio  (see https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/object-position)

```tsx
const style = {
	width: '100%',
	aspectRatio: '16 / 9',
	objectFit: 'cover',
	objectPosition: `${ focalPoint.x * 100 }% ${ focalPoint.y * 100 }%`,
};

return (
	<>
		<FocalPointPicker
			url={ url }
			value={ focalPoint }
			onDragStart={ setFocalPoint }
			onDrag={ setFocalPoint }
			onChange={ setFocalPoint }
		/>
		<img src={ url } alt="" style={ style } />
	</>
);
```
